server/models/Message.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require("mongoose");
+
+const messageSchema = new mongoose.Schema(
+  {
+    conversationId: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: "Conversation",
+      required: true,
+    },
+    senderId: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: "user",
+    },
+    message: {
+      type: String,
+      required: true,
+    },
+  },
+  { timestamps: true }
+);
+
+module.exports = Message = mongoose.model("message", messageSchema);
```

Comment:
This variable `Message` is not required if you didn't reference it anywhere in this file.
`module.exports = mongoose.model("Message", messageSchema);`

Reply:
Unused variable removed